staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -149,6 +149,12 @@ const (
 	//
 	// Allows label and field based indexes in apiserver watch cache to accelerate list operations.
 	SelectorIndex featuregate.Feature = "SelectorIndex"
+
+	// owner: @khenidak
+	// alpha: v1.15
+	//
+	// Enables ipv6 dual stack
+	IPv6DualStack featuregate.Feature = "IPv6DualStack"
```

Comment:
I think a local copy in cmd/cloud-controller-manager/app/core.go makes more sense unless there are other consumers that could benefit from a copy here.

Reply:
The tradeoff I made is future maintenance consideration VS importing extra dep. What would you think?